style.css

```diff
@@ -45,7 +50,7 @@ nav {
   align-items: flex-start;
   padding: 1px 24px;
   position: absolute;
-  width: 320px;
+  width: 100%;
```

Comment:
- [ ] Great job trying to make your page responsive 👍  but this width here has brought in an extra-wide section. I suggest removing it to make the page better responsive.
- Your design:
![d1](https://user-images.githubusercontent.com/85113567/149389925-750c2f40-b997-44bf-ba8f-bae30f2de87b.jpg)

Reply:
You can submit now, your site looks far better 👍